reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/CreatePotentialIssuesReportRuleProvider.java

```diff
@@ -29,7 +28,7 @@
 /**
  * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
  */
-@RuleMetadata(phase = ReportGenerationPhase.class)
+@RuleMetadata(disabled = true, phase = ReportGenerationPhase.class)
```

Comment:
@OndraZizka are you really sure with that? I tried this commit and it fails with the same failure. Master branch is on the other side passing.